### DISCUSSION TITLE
New kid on the block

### DISCUSSION THREAD

**Discussion by Danik1601** [iongion/container-desktop#92] [Show and tell]
Seems like the guys who make Podman create their own cross platform Desktop Companion with multi-engine support. What's our plan, general?
https://github.com/containers/podman-desktop

**Comment by Danik1601** [comment]
I created this discussion to let you know about the official project (it's hard to discover since if you Google "Podman Companion" all the links lead to your project). But then I discovered that [you are aware](https://www.reddit.com/r/podman/comments/rb6b6g/podman_desktop_companion_gui/ht7qnn1?utm_medium=android_app&utm_source=share&context=3) about it so I shouldn't have started the discussion in the first place 😅.
Sorry for fludding though.

**Reply by iongion** [reply]
Actually I was not up until last week, I was aware of the Swift Apple only thingy of Ashley, but nothing else, thanks for the heads-up.
What will happen, will definitely happen :)
